Blackfynn/data-portal
# Browse pagination #18awen

# Description

The purpose of this PR is to add pagination to the browse page.


## Type of change


# How Has This Been Tested?

- Go to the [browse page](http://127.0.0.1:5000/browse/#/)
- Pick "Files" from the search dropdown, and click on the "View Results" button
- Table should load showing search results, and pagination under should be display amount of pages/current page
- Use the pagination to ensure that pagination is working
- Search for the terms `.mat`
- Use the pagination to ensure that pagination is working
- Go to the last page of the pagination
- Pick "Datasets" from the search dropdown, clear the search terms, and click on the "View Results" button
- Datasets results should be showing results from page 1 (you can use your network tab to verify)

# Checklist:
